docs/modules/ROOT/pages/nre.adoc

```diff
@@ -0,0 +1,32 @@
+= NRE and Scripting
+
+Nile provides a Runtime Environment for scripting that you can use for multiple tasks. The Script is any python file implementing an asynchronous `run(nre)` function to receive a NileRuntimeEnvironment object exposing Nile's scripting API.
+
+== Running a script
+
+. Create a file named *test_script.py* in the root directory of your project.
++
+[,python]
+----
+async def run(nre):
+    accounts = await nre.get_accounts(predeployed=True)
```

Comment:
a short note for what predeployed means could be helpful for the newcomers

Reply:
I added a TIP pointing to the entry in the API Reference.